deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/AbstractLSTM.java

```diff
@@ -1,15 +1,12 @@
 /*******************************************************************************
  * Copyright (c) 2015-2018 Skymind, Inc.
  *
- * This program and the accompanying materials are made available under the
- * terms of the Apache License, Version 2.0 which is available at
- * https://www.apache.org/licenses/LICENSE-2.0.
+ * This program and the accompanying materials are made available under the terms of the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
```

Comment:
These licence headers should not be modified at all.
I realize the content is the same, but there's no reason to change it in just a couple of classes. It should stay consistent with the content of all other files.

Reply:
FYI, that's probably the formatter plugin doing this. Until we fix this, we should probably not recommend running it...